chapters/infrastructure/digital-rewards/reading/README.md

```diff
@@ -3,10 +3,38 @@
+We do not discuss the efectiveness of reward-based learning techniques, we simply provide the tools to implement such a system if an educator chooses to use it.
 
-goals of using digital rewards
+Rewards may be used at different steps in the learning process:
 
-infrastructure for digital rewards
+- when a learner correctly completes a designated teaching activity: a drill or set of drills, an assignment, an exam etc.
+- when a learner contributes content to the existing repo
+- when a learner helps his peers (by publicly answering forum questions or helping them during practical sessions)
 
-cryptocurrency
+Choosing the items for which rewards are offered is a subjective decision depending on personal preference and cultural background.
+However, from a usability perspective, there are two categories of rewards:
+
+- those that can be awarded automatically (e.g. a learner solves a multiple choice question)
+- those that are awarded manually (e.g. a learner has answered a verbal question during a lecture)
+
+To account for these situations, we have developed an infrastructure that uses an educational cryptocurrency that may be awarded automatically or manually.
+We use the [SmileyCoin](https://smileycoin.work/) educational cryptocurrency for that purpose.
+SmileyCoins are publicly traded and are worth close to nothing in terms of fiat currency.
+However, by offering certain prizes for specific SmileyCoin amounts, we can incentivize learners to self-study or to contribute to our repositories (as a mean of learning).
+The prizes that are being offered may vary, depending on the available resources of the entity that uses the course.
+SmileyCoins that are to be used as rewards for learners may be obtained by simply buying them (1.000.000 Smileys equal ~50 euros) or, in case that is not possible, by asking the [SmileyCoin Fund](https://smileyco.in/#/details) for a donation.
+
+To use the system, the following steps need to be taken:
+
+- the awarding entity sets up [a wallet](https://wallet.smileyco.in/?network=smileycoin) and deposits some smiley in there.
+- the potential receivers also create a wallet.
+- once an activity for which smiley coins are awarded is completed, the awarding entity sends an amount of smiley coins to the receiver.
+
+For the awarding entity to be able to send the necessary smiley coins, it is required that the receivers wallet address is known.
+As a consequence, the receiver needs to publicize his wallet address.
+As such, for smileys that are awarded automatically, the receiver needs to have an account that is associated with his wallet on the platform where the activity is performed (either a github account for contributions or a platform account on the hosting service - such as TutorWeb).
+For smiley that are awarded manually, the receiver may simply communicate the receiving wallet address.
+
+Currently, we have implemented some scripts that make it easy to manually send smileycoin rewards.
```

Comment:
You should add links to them here [1], and they ought to be detailed in the Infrastructure chapter.

[1] https://github.com/Jokeswar/smileycoin-api/wiki/Teaching-Assistan-Handbook